test/testdata/cfg/retry_nested.rb.cfg.exp

```diff
@@ -35,7 +35,7 @@ subgraph "cluster_::Object#main" {
 
     "bb::Object#main_4" -> "bb::Object#main_5" [style="bold"];
     "bb::Object#main_5" [
-        label = "block[id=5, rubyBlockId=0](<returnMethodTemp>$2: NilClass, <self>: Object, <gotoDeadTemp>$33: NilClass, try: Integer(0))\l<exceptionValue>$8: T.untyped = <unanalyzable>\l<exceptionValue>$8: T.untyped\l"
+        label = "block[id=5, rubyBlockId=1](<returnMethodTemp>$2: NilClass, <self>: Object, <gotoDeadTemp>$33: NilClass, try: Integer(0))\l<exceptionValue>$8: T.untyped = <unanalyzable>\l<exceptionValue>$8: T.untyped\l"
```

Comment:
am i reading this diff right that this was an intentional change? that rubyBlockId was incorrect here before?

Reply:
Yes, this was intentional. The `rubyBlockId` was incorrect if you used conditionals within an exception handling block. With this change, the correct block id is plumbed through.